itests/standalone/basic/src/test/java/org/wildfly/camel/test/hipchat/HipchatConsumerIntegrationTest.java

```diff
@@ -0,0 +1,207 @@
+            }
+        });
+        return context;
+    }
+
+    @Test
+    public void sendInOnly() throws Exception {
+        CamelContext camelctx = createCamelContext();
+
+        MockEndpoint result = camelctx.getEndpoint("mock:result", MockEndpoint.class);
+        result.expectedMessageCount(1);
+
+        camelctx.start();
+        try {
+            String expectedResponse = "{\n" + "  \"items\" : [\n" + "    {\n" //
+                    + "      \"date\" : \"2015-01-19T22:07:11.030740+00:00\",\n" //
+                    + "      \"from\" : {\n" //
+                    + "        \"id\" : 1647095,\n" //
+                    + "        \"links\" : {\n" //
+                    + "          \"self\" : \"https://api.hipchat.com/v2/user/1647095\"\n" //
+                    + "        },\n" //
+                    + "        \"mention_name\" : \"notifier\",\n" //
+                    + "        \"name\" : \"Message Notifier\"\n" //
+                    + "      },\n" //
+                    + "      \"id\" : \"6567c6f7-7c1b-43cf-bed0-792b1d092919\",\n" //
+                    + "      \"mentions\" : [ ],\n" //
+                    + "      \"message\" : \"Unit test Alert\",\n" //
+                    + "      \"type\" : \"message\"\n" //
+                    + "    }\n" //
+                    + "  ],\n" //
+                    + "  \"links\" : {\n" //
+                    + "    \"self\" : \"https://api.hipchat.com/v2/user/%40ShreyasPurohit/history/latest\"\n" //
+                    + "  },\n" //
+                    + "  \"maxResults\" : 1,\n" //
+                    + "  \"startIndex\" : 0\n" //
+                    + "}";
+            HttpEntity mockHttpEntity = mock(HttpEntity.class);
+            when(mockHttpEntity.getContent())
+                    .thenReturn(new ByteArrayInputStream(expectedResponse.getBytes(StandardCharsets.UTF_8)));
+            when(closeableHttpResponse.getEntity()).thenReturn(mockHttpEntity);
+            when(closeableHttpResponse.getStatusLine())
+                    .thenReturn(new BasicStatusLine(new ProtocolVersion("HTTP", 1, 1), 200, ""));
+
+            result.assertIsSatisfied();
+
+            assertCommonResultExchange(result.getExchanges().get(0));
+        } finally {
+            camelctx.stop();
+        }
+    }
+
+    private void assertCommonResultExchange(Exchange resultExchange) {
+        HipchatProducerIntegrationTest.assertIsInstanceOf(String.class, resultExchange.getIn().getBody());
+        Assert.assertEquals("Unit test Alert", resultExchange.getIn().getBody(String.class));
+        Assert.assertEquals("@AUser", resultExchange.getIn().getHeader(HipchatConstants.FROM_USER));
+        Assert.assertEquals("2015-01-19T22:07:11.030740+00:00",
+                resultExchange.getIn().getHeader(HipchatConstants.MESSAGE_DATE));
+        Assert.assertNotNull(resultExchange.getIn().getHeader(HipchatConstants.FROM_USER_RESPONSE_STATUS));
+    }
+
+    // TODO
```

Comment:
Guessing this TODO comment is superfluous?

Reply:
> Guessing this TODO comment is superfluous?

Yes, it is. I have fixed it locally, but I do not want to push before https://github.com/wildfly-extras/wildfly-camel/pull/2321 is merged so that I do not trigger a new build with the old SNS test.

Reply:
Fixed in fe9e51d